server/app.py

```diff
@@ -1,19 +1,25 @@
 from flask import Flask
+from flask_jwt_extended import JWTManager
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
 
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
 from config import Config
 
-app = Flask(__name__)
+app = Flask(__name__.split('.')[0])
 app.config.from_object(Config)
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 
 
 db = SQLAlchemy(app)
+jwt = JWTManager(app)
 migrate = Migrate(app,db)
 from models import User
+# authentication_handler imports db from this very file.
+# Is there a better file structure for this?
+from api.authentication_handler import authentication_handler
```

Comment:
For both these and the below comment, the `app.py` for Flask does require us to have imports in a non-standardised ways. The `api` and `models` need to import from app, and putting the imports below the `db = SQLAlchemy(app)` is a requirement, else it would throw up errors due to circular depedencies.